RasiakMykola/HW4/TASK_1.py

```diff
@@ -0,0 +1,15 @@
+
+# 04.1 PRACTICAL TASKS / [GITHUB]
+
+
+# Task1. "Temperature Converter"
+celsius = float(input("\nEnter the temperature in Celsius: "))
+                
+if celsius < -273.15 :
+    
+    print("\nError: Temperature below absolute zero (-273.15°C)")
+
+else :
+    
+    F = (celsius*9/5)+32
```

Comment:
use more meaningful names for variables like this

celsius = float(input("Enter the temperature in Celsius: "))

absolute_zero_celsius = -273.15

if celsius < absolute_zero_celsius:
    print("Error: Temperature below absolute zero ({:.2f}°C)".format(absolute_zero_celsius))
else:
    fahrenheit = (celsius * 9/5) + 32
    print("{:.2f}°C is equivalent to {:.2f}°F".format(celsius, fahrenheit))